release-engineering/packaging.md

```diff
@@ -70,7 +70,7 @@ git clone https://github.com/kubernetes/release.git
 cd release
 ```
 
-**IMPORTANT: You must checkout a `kubernetes/release` tag >= [`v0.3.3`](https://github.com/kubernetes/release/releases/tag/v0.3.3) to address a [CVE for CNI plugins](https://github.com/kubernetes/kubernetes/issues/91507).**
+You must checkout `kubernetes/release` -> [`master`](https://github.com/kubernetes/release.git)
```

Comment:
https://github.com/kubernetes/sig-release/pull/1151

discussion there seems relevant.
maybe we want to go ahead and migrate it to the other repo next.

Reply:
Good point